config.json

```diff
@@ -13,6 +13,6 @@
             "device_info_service": 1,
             "security_level": "SECURITY_MODE_ENCRYPTION_NO_MITM"
         },
-        "gatt_table_size": "0x600"
+        "gatt_table_size": "0x500"
```

Comment:
I believe this creates a little more room for the shared stack+heap space. Without this change, the `snprintf` below caused a memory error.